Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the Include Granted Scopes to determine if this authorization request should use
+        /// incremental authorization. If this is provided with the value <c>true</c>, and the authorization 

```

Comment:
Removed everything starting at and following "If this is provided..." and added in
*For additional details see https://developers.google.com/+/web/api/rest/oauth#incremental-auth.*

> and a .NET tag for the link)" after incremental authorization.

Sorry, not sure I follow. Do you mean something like [this](http://stackoverflow.com/a/6960573/792032)?